src/components/Blogs/BlogItem.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styles from "./BlogItem.module.scss";
+const BlogItem = ({ img, bhead, btitle, btext, bdimg, bwriter, bdate }) => {
```

Comment:
ideally, the prop names should be camelCase.

bdimage -> bdImage
bdate -> bDate
etc

Reply:
solved